### ISSUE TITLE
Load Bakugan from JSON file

### ISSUE THREAD

**Issue by Matteas-Eden** [Matteas-Eden/bakugan-rerolled#7]
Instead of creating Bakugan within the source code itself, I want to move to a system where they can be loaded from an external file. In this case, JSON seems appropriate. 

So instead of this:
```java
Bakugan saurus = new Bakugan("Saurus", dan, 270, Attribute.Pyrus);
```

I want to store the data somewhat like this:
```
{
	"id":34,
	"name":"Saurus",
	"attribute":"Pyrus",
	"gpower":270,
	"special":{
		"effects":[],
		"conditions":[]
	}
},
```

*Note: The 'special' parameter is a later issue in regards to character Bakugan with special abilities*

**Comment by Matteas-Eden** [comment]
Addendum: This may involve the use of a custom JSON parser.

**Comment by Matteas-Eden** [comment]
Resolved by 83a3a2b8b64bb236e3ca7f78dd71280b27a87fee
